mono/profiler/mono-profiler-log.c

```diff
@@ -3838,6 +3732,32 @@ init_suppressed_assemblies (void)
 }
 
 static void
+parse_cov_filter_file (GPtrArray *filters, const char *file)
+{
+	FILE *filter_file;
+	char *line, *content;
+
+	if (filters == NULL)
```

Comment:
`filters` are already initialized in line 4614. Furthermore, it's a function-local variable, so the assigned array will be lost once we leave the function